src/index.js

```diff
@@ -84,17 +111,22 @@ const printList = () => {
       printList();
     });
   });
+  checkCompleted();
+  setCheckboxListener();
 };
 
 printList();
 
+// console.log(setCheckboxListener);
+
 const addNewTask = () => {
   toDoList = getToDoList();
   const newIndex = toDoList.length + 1;
   const task = new Task(newTaskInput.value, newIndex);
   task.Add();
   newTaskInput.value = '';
   printList();
+  // checkCompleted();
 };
 
 // Add new Task
```

Comment:
- [ ] Please know that adding a new task isn't working at all kindly fix the issue as I wasn't able to test anything.